features/10-sync-for-science.feature

```diff
@@ -1,22 +1,19 @@
-@s4s
+@use.with_use_case=ehr @s4s
```

Comment:
Technically we want two more blocks like this: one `@use.with_use_case=financial` and one  `@use.with_use_case=imaging`. Feel free to fix here or just add an issue to address later!

Reply:
Good point. I'll see what we can fit in one feature file.

https://github.com/sync-for-science/tracking/issues/138